odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -21,14 +21,160 @@
 standard_library.install_aliases()
 
 import numpy as np
+import scipy as sp
 from odl.discr import ResizingOperator
 from odl.trafos import FourierTransform, PYFFTW_AVAILABLE
+from odl.tomo.util.utility import perpendicular_vector
 
 
-__all__ = ('fbp_op',)
+__all__ = ('fbp_op', 'tam_danielson_window')
 
 
-def fbp_op(ray_trafo, padding=True):
+def _axis_in_detector(geometry):
+    """A vector in the detector plane that points along the rotation axis."""
+    du = geometry.det_init_axes[0]
+    dv = geometry.det_init_axes[1]
+    axis = geometry.axis
+    c = np.array([np.vdot(axis, du), np.vdot(axis, dv)])
+    cnorm = np.linalg.norm(c)
+
+    # Check for numerical errors
+    assert cnorm != 0
+
+    return c / cnorm
+
+
+def _rotation_direction_in_detector(geometry):
+    """A vector in the detector plane that points in the rotation direction."""
+    du = geometry.det_init_axes[0]
+    dv = geometry.det_init_axes[1]
+    axis = geometry.axis
+    det_normal = np.cross(du, dv)
+    rot_dir = np.cross(axis, det_normal)
+    c = np.array([np.vdot(rot_dir, du), np.vdot(rot_dir, dv)])
```

Comment:
The function could be useful for other situations as well where no absolute value is taken, so 👍 for fixing it.